scripts/gen_FCHL19_rep.py

```diff
@@ -0,0 +1,161 @@
+        print("read xyz file used for a dictionary:", dictxyz, ", a total of",
+              ndictframes, "frames")
+
+    frames = dictframes + fframes
+    nframes = len(frames)
+    global_species = []
+    for frame in frames:
+        global_species.extend(frame.get_atomic_numbers())
+        if not periodic:
+            frame.set_pbc([False, False, False])
+    global_species = np.unique(global_species)
+    print("a total of", nframes, "frames, with elements: ", global_species)
+
+
+    foutput = prefix + "_FCHL19" 
+    desc_name = "FCHL19" 
+
+    # prepare for the output
+    if os.path.isfile(foutput + ".xyz"): os.rename(foutput + ".xyz", "bck." + foutput + ".xyz")
+    if os.path.isfile(foutput + ".desc"): os.rename(foutput + ".desc", "bck." + foutput + ".desc")
+
+    for i, frame in enumerate(frames):
+        #print(i,frame)
+        #print(frame.get_positions())
+        rep = repr_wrapper(frame,global_species,periodic,nRs2=nRs2, nRs3=nRs3, nFourier=nFourier,
+                                                eta2=eta2, eta3=eta3, zeta=zeta, 
+                                                rcut=rcut, acut=acut,
+                                                two_body_decay=two_body_decay, three_body_decay=three_body_decay,
+                                                three_body_weight=three_body_weight)
+        print(rep.shape,rep.mean(axis=0).shape)                              
+        frame.info[desc_name] = rep.mean(axis=0)
+
+        # save
+        if output == 'matrix':
+            with open(foutput + ".desc", "ab") as f:
+                np.savetxt(f, frame.info[desc_name][None])
+            if peratom or nframes == 1:
+                with open(foutput + ".atomic-desc", "ab") as fatomic:
+                    np.savetxt(fatomic, rep)
+        elif output == 'xyz':
+            # output per-atom info
+            if peratom:
+                frame.new_array(desc_name, rep)
+            # write xyze
+            write(foutput + ".xyz", frame, append=True)
+        else:
+            raise ValueError('Cannot find the output format')
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-fxyz', type=str, required=True, help='Location of xyz file')
+    parser.add_argument('-fdict', type=str, default='none', help='Location of xyz file '
+                                                                 'that is used for a dictionary')
+    parser.add_argument('--prefix', type=str, default='ASAP', help='Filename prefix')
+    parser.add_argument('--output', type=str, default='xyz', help='The format for output files ([xyz], [matrix])')
+    parser.add_argument('--peratom', type=str2bool, nargs='?', const=True, default=False,
+                        help='Do you want to output per atom representations for multiple frames (True/False)?')
+                        
+    parser.add_argument('--nRs2', type=int, default=24, help='number radial bins in the two-body term')
```

Comment:
Could we add these descriptions for the parameters to the function docstrings above?